pkg/services/ngalert/state/historian/loki_http.go

```diff
@@ -11,25 +11,36 @@ import (
 
 const defaultClientTimeout = 30 * time.Second
 
+type LokiConfig struct {
+	Url            *url.URL
+	TenantID       string
+	TenantPassword string
+}
+
 type httpLokiClient struct {
 	client http.Client
-	url    *url.URL
+	cfg    LokiConfig
 	log    log.Logger
 }
 
-func newLokiClient(u *url.URL, logger log.Logger) *httpLokiClient {
+func newLokiClient(cfg LokiConfig, logger log.Logger) *httpLokiClient {
 	return &httpLokiClient{
 		client: http.Client{
 			Timeout: defaultClientTimeout,
 		},
-		url: u,
+		cfg: cfg,
 		log: logger.New("protocol", "http"),
 	}
 }
 
 func (c *httpLokiClient) ping() error {
-	uri := c.url.JoinPath("/loki/api/v1/status/buildinfo")
+	uri := c.cfg.Url.JoinPath("/loki/api/v1/labels")
 	req, err := http.NewRequest(http.MethodGet, uri.String(), nil)
+
+	if c.cfg.TenantID != "" && c.cfg.TenantPassword != "" {
```

Comment:
```suggestion
	if c.cfg.TenantID != "" || c.cfg.TenantPassword != "" {
```
So that empty is allowed